tests/e2e/navigate_chromium_cdp.mjs

```
@@ -0,0 +1,106 @@
+  // Browser-level Storage.getCookies hangs in several macOS Chromium forks,
+  // even after Target.createTarget succeeds. Give the network request time to
+  // finish and let the Python harness verify the persisted SQLite row instead.
+  await new Promise((resolve) => setTimeout(resolve, 2_000));
+  console.log(`native CDP navigation requested ${url}`);
+} finally {
+  // A protocol-level close gives the native process a chance to checkpoint
+  // its persistent cookie database before the extractor opens it.
+  try {
+    await send("Browser.close");
+  } catch (error) {
+    if (socket.readyState === WebSocket.OPEN) throw error;
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)